pkg/image/image.go

```diff
@@ -61,3 +61,45 @@ func ParseReference(ref string) (imagesource.TypedImageReference, error) {
 	}
 	return imagesource.TypedImageReference{Ref: dst, Type: dstType}, nil
 }
+
+// parseImageName returns the registry, organisation, repository, tag and digest
+// from the imageName.
+// It can handle both remote and local images.
+func ParseImageReference(imageName string) (string, string, string, string, string) {
```

Comment:
Since this is an exported function now, it may be worth it to have some validation for the image string to ensure it is an image reference. If a string was passed that was not in the correct format, it could yield some unexpected results.